services/parser-database/tests/test_connector.py

```diff
@@ -35,6 +35,18 @@
 }
 
 
+keywords_mock = ["forzoso", "bicicleta", "usar", "casco"]
```

Comment:
These can be just local variables to the one test case. 

Also [nit] typically mocks refer to when you fake out the return of a service or function because you can't call the actual function, but in this case we are not circumventing anything, so you don't have to include `mock` in the name. These are just normal, valid inputs to this function.